lib/contact_us.rb

```diff
@@ -19,6 +19,10 @@ module ContactUs
   # URL after a successful submission
   mattr_accessor :success_redirect
 
+  # Configure parent mailer
+  mattr_accessor :parent_mailer
+  @@parent_mailer = 'ActionMailer::Base'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.